linetools/spectralline.py

```diff
@@ -482,7 +487,11 @@ def fill_data(self, trans, linelist=None, closest=False, verbose=True):
 
         # Data
         newline = llist[trans]
-        self.data.update(newline)
+        try:
```

Comment:
Please add commented line explaining that newline is expected to be a LineList dict() output.